### ISSUE TITLE
JavaScript Errors appear when ACF rules for Taxonomy set equal to Product Categories. Results in inability to add categories

### ISSUE THREAD

**Issue by Pcosta88** [Yoast/yoast-acf-analysis#13]
### What did you expect to happen?

No errors would be generated 
### What happened instead?

JavaScript errors were generated
### How can we reproduce this behavior?

**1.Install Advanced Custom Fields (Pro or free)**

**2.Install Yoast SEO Premium** 

**3.Install Yoast ACF SEO Analysis**

**4.Make a field set to taxonomy for product categories. Use WYSIWYG editor (also happens with text editor)**

![screen shot 2016-09-19 at 10 28 16 am](https://cloud.githubusercontent.com/assets/8238074/18635678/cd19c652-7e53-11e6-8f32-6fc2c400694f.png)

**5.Create a product_cat**

![screen shot 2016-09-19 at 10 29 37 am](https://cloud.githubusercontent.com/assets/8238074/18635769/254d8390-7e54-11e6-9608-ac60e6e6ea2e.png)

**5.See briefly that JavaScript errors are thrown out**

```
TypeError: Cannot read property 'replace' of undefined
 /wp-content/plugins/wordpress-seo-premium/premium/assets/js/dist/wp-seo-premium-quickedit-notification-352.min.js?ver=3.5.2:1

```

**6.See that product cat is still created**

![screen shot 2016-09-19 at 10 30 00 am](https://cloud.githubusercontent.com/assets/8238074/18635774/2de2144e-7e54-11e6-92c3-27260e3041d0.png)

**7.Switch to Advanced Custom Fields Pro while using Yoast SEO Premium**

**8.Add a field, create product cat, see that JavaScript errors are thrown out as above. See also that the product cat fails to be added.**

![screen shot 2016-09-19 at 10 33 09 am](https://cloud.githubusercontent.com/assets/8238074/18635864/9251a1ec-7e54-11e6-8aea-109a931a3026.png)

**9.Keep Advanced Custom Fields Pro active while using Yoast SEO (free). See that no JavaScript errors are thrown out AND product cat is added as expected.** 

**10.Advanced Custom Fields (free one) and Yoast SEO (free) do not result in the JavaScript Errors**

Can you provide a link to a page which shows this issue?
### Technical info
- WordPress version: 4.6.1
- Yoast SEO (free): 3.5
- Yoast SEO Premium (paid): 3.5.2
- Advanced Custom Fields (PRO): 5.4.5
  *WooCommerce 2.6.4
  *Advanced Custom Fields (free): 4.4.9
- Yoast SEO ACF Analysis: 1.2.0

**Comment by Pcosta88** [comment]
Please inform the customer of conversation # 147154 when this conversation has been closed.

**Comment by mikakaltoft** [comment]
@Pcosta88 What is status on this issue?

**Comment by Pcosta88** [comment]
@mikakaltoft  The issue was also reported in another repository here:https://github.com/Yoast/wordpress-seo-premium/issues/770. On the right-side you can view the progress of the issue under the title "Milestone".

**Comment by kraftner** [comment]
I feel like this is an issue with Yoast SEO itself and not this plugin. @Pcosta88 can you check with the premium repo to see if this might already be resolved there so we can close this issue?

**Comment by Pcosta88** [comment]
@kraftner Confirmed: https://github.com/Yoast/wordpress-seo-premium/issues/770

**Comment by kraftner** [comment]
@Pcosta88 Somehow this slipped through - by confirmed you mean this issue is now fixed and can be closed?

**Comment by Pcosta88** [comment]
@kraftner Correct, The premium repo is closed. This issue was resolved in Yoast SEO (free or premium) v4.6

**Comment by kraftner** [comment]
Thx! Hence closing this.
